code/modules/cargo/exports/sheets.dm

```diff
@@ -83,6 +83,22 @@
 	unit_name = "wood plank"
 	export_types = list(/obj/item/stack/sheet/mineral/wood)
 
+/datum/export/stack/plastic
```

Comment:
Plastic is not scarce, since it can be produced in infinite quantities with roundstart chemistry/plumbing machines, so 50 credits per sheet is far too high. having it on the level of cloth or metal would be healthier.

Reply:
Yep, remove potentially infinite sources of cash that can be 100% automatized in production.